server/controllers/bookingRequest.js

```diff
@@ -0,0 +1,81 @@
+const asyncHandler = require("express-async-handler");
+const User = require("../models/User");
+const BookingRequest = require("../models/BookingRequest");
+
+// @route POST /booking-requests/
+// @desc Create a new request
+// @access Private
+
+exports.createBookingRequest = asyncHandler(async (req, res, next) => {
+  const userId = req.user.id;
+  const user = await User.findById(userId);
+  const { sitterId, start, end, description } = req.body;
+
+  const sitter = await User.findById(sitterId);
+  if (sitter) {
+    const bookingRequest = await BookingRequest.create({
+      userId: user,
+      sitterId: sitter,
+      start,
+      end,
+      description,
+    });
+
+    res.status(201).json({
+      success: {
+        bookingRequest: {
+          request_id: bookingRequest._id,
+          userId: bookingRequest.userId._id,
+          sitterId: bookingRequest.sitterId._id,
+          start: bookingRequest.start,
+          end: bookingRequest.end,
+          description: bookingRequest.description,
+        },
+      },
+    });
+  } else {
+    res.status(400);
```

Comment:
```suggestion
    res.status(500);
```
This will be a server error if request creation has failed